File: BKabanApi/BKabanApi/Models/BoardModel.cs
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;

namespace BKabanApi.Models
{
    public class BoardModel
    {
        public int? Id { get; set; }

        [MaxLength(100)]
        public string Name { get; set; }

        public IEnumerable<ColumnModel> Columns { get; set; }
    }
}

File: BKabanApiJava/src/main/java/edu/bkaban/models/board/BoardModel.java
package edu.bkaban.models.board;

import edu.bkaban.models.column.ColumnModel;
import org.hibernate.validator.constraints.Length;

import java.util.List;

public class BoardModel {

    private Integer Id;

    @Length(max = 100)
    private String Name;

    private List<ColumnModel> Columns;

    public Integer getId() {
        return Id;
    }

    public void setId(Integer id) {
        Id = id;
    }

    public String getName() {
        return Name;
    }

    public void setName(String name) {
        Name = name;
    }

    public List<ColumnModel> getColumns() {
        return Columns;
    }

    public void setColumns(List<ColumnModel> columns) {
        Columns = columns;
    }

    public BoardModel(Integer id, String name) {
        Id = id;
        Name = name;
    }

    public BoardModel() {}
}
